src/common/components/Item/index.js

```diff
@@ -79,11 +79,16 @@ const Item = ({
   // $FlowFixMe
   const error = item && item.error;
   const itemLoaded = !error && !!Object.keys(item).length;
+  const skinLoaded = !error && !!Object.keys(skin).length;
 
   let tooltipData;
 
   if (error) {
     tooltipData = error;
+  } else if (skinLoaded) {
```

Comment:
this will end up taking priority of a skin over an item.

maybe we should utilise `tooltipType` (and give it a default prop of 'items').

if error -> error tooltip
if tooltiptype = item, -> item tooltip
if tooltiptype = skin, -> skin tooltip

would be good to test this logic too, since CI is failing because coverage dropped :)..

Reply:
Where is tooltipType defined? It is always undefined and does not seem reliable for making this check.

https://github.com/madou/armory-app/pull/353/files/8872c7b56fc3aa8d076fde920f596ba9ef65a180#diff-f1fbdef86345ad9836efc20bb277f849R66

Reply:
its just a prop passed in. so we would inside `Gw2Skin` component, pass the prop `tooltipType="skin"`.

:)